sylt-parser/src/parser.rs

```diff
@@ -539,33 +541,33 @@ pub fn parse_type<'t>(ctx: Context<'t>) -> ParseResult<'t, Type> {
             let mut ctx = ctx.skip(1);
             let mut types = Vec::new();
             // Tuples may (and probably will) contain multiple types.
+            let mut is_tuple = matches!(ctx.token(), T::Comma | T::RightParen);
```

Comment:
It shouldn't be able to start with a comma right?
Perhaps skip it at the end of the loop.

Reply:
It now matches the tuple syntax for values. As far as I know, this is correct.